packages/composables/src/useProduct.ts

```diff
@@ -31,10 +30,7 @@ export function useProduct(
   configurator?: Ref<Schemas["PropertyGroup"][]> | Schemas["PropertyGroup"][],
 ): UseProductReturn {
   const _product = useContext("product", { context: product });
-  if (!_product.value) {
-    // TODO link docs with composables context usage
-    throw new ContextError("Product");
-  }
```

Comment:
why is this removed?
If product is not provided with param or context it would cause runtime errors